index.js

```diff
@@ -73,25 +89,24 @@ module.exports = ({ types: t }) => {
 
         if(!t.isArrayExpression(dependencyList) && !factory) return;
 
-        let injectsModuleOrExports = false;
         const isFunctionFactory = t.isFunctionExpression(factory);
         const requireExpressions = [];
+        // Order is important here for the simplified commonjs wrapper
+        const keywords = [REQUIRE, EXPORTS, MODULE];
 
         if(dependencyList) {
-          dependencyList.elements.forEach((el, i) => {
-            if(t.isStringLiteral(el)) {
-              switch(el.value) {
-                case REQUIRE:
-                  return;
-                case MODULE:
-                case EXPORTS:
-                  injectsModuleOrExports = true;
-                  return;
-              }
-            }
-            const paramName = isFunctionFactory && factory.params[i];
-            requireExpressions.push(createRequireExpression(el, paramName));
+          const dependencyParameterPairs = zip(
+            dependencyList.elements,
+            isFunctionFactory ? factory.params : []
+          );
+
+          const explicitRequires = dependencyParameterPairs.filter(([dependency]) => {
+            return !t.isStringLiteral(dependency) || keywords.indexOf(dependency.value) === -1;
```

Comment:
Using `indexOf` here instead of `includes` for [Node.js 4.x compatibility](http://node.green/#ES2016-features-Array-prototype-includes-Array-prototype-includes).